src/providers/ldap/sdap_async_initgroups_ad.c

```diff
@@ -1611,9 +1613,22 @@ sdap_ad_tokengroups_initgroups_send(TALLOC_CTX *mem_ctx,
      * to avoid having to transfer and retain members when the fake
      * tokengroups object without name is replaced by the full group object
      */
+    use_shortcut = false;
     if (state->use_id_mapping
             && !IS_SUBDOMAIN(state->domain)
-            && state->domain->ignore_group_members == false) {
+            && !state->domain->ignore_group_members) {
```

Comment:
Why 'shortcut' isn't used if 'ignore_group_members = true'?

Reply:
Well, @sumit-bose pointed me to a comment above, but I have a hard times trying to understand it :-/

Reply:
From the comment right above:

> Additionally, as a temporary workaround until https://fedorahosted.org/sssd/ticket/2656 is fixed, we also fetch the group object if group members are ignored to avoid having to transfer and retain members when the fake tokengroups object without name is replaced by the full group object.

That ticket has been closed as "not a bug", so I guess we need to keep this workaround.